ui/js/controllers/bugfiler.js

```diff
@@ -166,6 +168,11 @@ treeherder.controller('BugFilerCtrl', [
                 descriptionStrings += $scope.modalComment;
             }
 
+            var keywords = "";
+            if($scope.isIntermittent) {
+                keywords += "intermittent-failure";
```

Comment:
Yeah, it was a serious case of premature optimization. :) 
I'll fix it up tomorrow and reflag you for review.